modules/tinymce/src/plugins/image/test/ts/browser/ImagePluginTest.ts

```diff
@@ -218,4 +225,16 @@ describe('browser.tinymce.plugins.image.ImagePluginTest', () => {
       }
     )
   );
+
+  it('TINY-3463: Ensure initial toolbar button state shows correctly', async () => {
+    const editor = hook.editor();
+    editor.setContent('<p>Content</p><p><img src="image.png"></p>');
+
+    TinySelections.setCursor(editor, [ 0, 0 ], 4);
+    await TinyUiActions.pWaitForUi(editor, '.tox-tbtn:not(.tox-tbtn--enabled)');
```

Comment:
Should we add `[title="..."]` or `:contains("...")` to these selectors? I'm a little worried about another toolbar button being added to the hook above without seeing this, and all of these tests no longer checking the right thing

Reply:
We don't anywhere else here or in other tests where we are guaranteed to only have the one toolbar button, so why do you want it in this single place?

Reply:
Sounds like this is maybe something we should discuss and change how we handle this overall? I'm happy to change it either way, but at this stage there's quite a number of places where we don't do this.

Reply:
Have added the tooltips to the CSS selectors.